src/openepd/model/epd.py

```diff
@@ -271,3 +271,10 @@ def validate_compliance(cls, v):
         if v is None:
             return []
         return v
+
+    @pyd.validator("product_image", always=True, pre=True)
+    def validate_product_image(cls, v):
+        """Handle correctly validation for product image field.."""
+        if v is not None and isinstance(v, str) and v.startswith("file://"):
```

Comment:
Remove `v is not None` because you are doing `is_instance` further.

Reply:
removed